server/report/report-generic.js

```diff
@@ -24,20 +26,19 @@ export default class {
     this.report.onStart(null, this.context);
     /* istanbul ignore else */
     if (childs.length === 0) {
-      this.res.setHeader('Content-type', 'text/plain');
-      this.res.statusCode = 500;
-      return this.res.end('{"type":"No coverage to export"}');
+      this.res.set('Content-type', 'text/plain');
+      this.res.status(500);
+      return this.res.json({'type':'No coverage to export'});
     }
 
     this.writeFile(childs);
-    this.res.end('{"type":"success"}');
+    this.res.json({'type':'success'});
   }
 
   writeFile(childs) {
     for (let i = 0; i < childs.length; i++) {
       // Remove the COVERAGE_APP_FOLDER from the filepath
-      childs[i].fileCoverage.data.path = childs[i].fileCoverage.data.path.replace(Conf.COVERAGE_APP_FOLDER, '');
-
+      // data.path = childs[i].fileCoverage.data.path.replace(Conf.COVERAGE_APP_FOLDER, '');
```

Comment:
Throws an error, since newer version of handlebars to not allow values derived from protoype due to prototype pollution, xss etc. so this line always throws an error. Updated the istanbul reporter but forgot to uncomment, since it was working anyway but this might get into conflict with custom Coverage paths?

You can try to uncomment and see what happens :see_no_evil:

Reply:
Okay it works now, I uncommented it back. Looks like the newer versions of istanbul reporters are now supporting this handlebars requirement.